GitUp/Application/Document.m

```diff
@@ -1173,7 +1173,7 @@ - (NSToolbarItem*)toolbar:(NSToolbar*)toolbar itemForItemIdentifier:(NSString*)i
 
 - (NSArray*)toolbarDefaultItemIdentifiers:(NSToolbar*)toolbar {
   if (_unifiedToolbar) {
-    return @[ kToolbarItem_Left, kToolbarItem_Title, NSToolbarSpaceItemIdentifier, kToolbarItem_Right ];
+    return @[ kToolbarItem_Left, kToolbarItem_Title, kToolbarItem_Right ];
```

Comment:
Why is this changing?

Reply:
I explained why in the commit message:

> An NSToolbarSpaceItemIdentifier-type toolbar item swallows mouse-ups,
> preventing a double click in the titlebar from zooming the window.
> Instead of using a space item, just add the padding to the right toolbar
> item's custom view.